documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -55,16 +52,26 @@ To specify a different root device, in the *Settings* section, click the Edit ic
 {project-short} uses the following format for disk location: `/dev/sd<disk_identifier><disk_partition>`. For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
 +
 If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.
+====
 
-. {rhv-short} source providers only (Optional):
+endif::[]
+ifdef::rhv[]
++
+[IMPORTANT]
+====
+Check the following {rhv-short}-specific items. All are optional, but need special attention:
 
 * *Preserving the CPU model of VMs that are migrated from {rhv-short}*: Generally, the CPU model (type) for {rhv-short} VMs is set at the cluster level, but it can be set at the VM level, which is called a custom CPU model.
 By default, {project-short} sets the CPU model on the destination cluster as follows: {project-short} preserves custom CPU settings for VMs that have them, but, for VMs without custom CPU settings, {project-short} does not set the CPU model. Instead, the CPU model is later set by {virt}.
 +
 To preserve the cluster-level CPU model of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. Toggle the *Whether to preserve the CPU model* switch, and then click *Save*.
+====
+endif::[]
 
-. If the plan is valid,
+. If your plan is valid:
 .. You can run the plan now by clicking *Start migration*.
-.. You can run the plan later by selecting it on the *Plans for virtualization* page and following the procedure in xref:running-migration-plan_mtv[Running a migration plan].
-+
+.. You can run the plan later by selecting it on the *Plans for virtualization* page and following the procedure in xref:running-migration-plan_{context}[Running a migration plan].
```

Comment:
Up to you...  I don't think you need "You can."

Reply:
@nunzy1 Here, I changed the lead-in.